data/ignition.template

```diff
@@ -16,15 +16,32 @@ systemd:
       contents: |
         [Unit]
         Description=Service to retrieve system uuid
+        Requires=var-lib-data.mount
+        After=var-lib-data.mount
         [Service]
         Type=oneshot
-        ExecStart=/bin/bash -c 'cat /sys/class/dmi/id/product_uuid > /var/home/core/.migration-planner/data/agent_id'
-        ExecStartPost=/bin/bash -c 'chown core:core /var/home/core/.migration-planner/data/agent_id'
+        ExecStart=/bin/bash -c 'cat /sys/class/dmi/id/product_uuid > /var/lib/data/agent_id'
+        ExecStartPost=/bin/bash -c 'chown -R core:core /var/lib/data'
         RemainAfterExit=true
         [Install]
         WantedBy=multi-user.target
+    - name: var-lib-data.mount
+      enabled: true
+      contents: |
+        [Mount]
+        What=/dev/disk/by-label/DATA
+        Where=/var/lib/data
+        Type=ext4
+        [Install]
+        WantedBy=local-fs.target
 
 storage:
+  filesystems:
+    - path: /var/lib/data
+      device: /dev/sda
```

Comment:
I think it's safe for now, but maybe in future we should improve it to use ID?

Reply:
good point, future improvement!